deploy/function-call/fc-create-sdk.js

```diff
@@ -0,0 +1,40 @@
+const { FUNDING_ACCOUNT_ID, FUNDER_INFO, NETWORK_ID, NUM_KEYS, DROP_METADATA, DROP_CONFIG, KEYPOM_CONTRACT, FC_DATA, DEPOSIT_PER_USE_NEAR} = require("./configurations");
+
+const path = require("path");
+const homedir = require("os").homedir();
+const { writeFile, mkdir, readFile } = require('fs/promises');
+const { initKeypom, createDrop, getDrops } = require("keypom-js");
+
+//funder is account to sign txns, defaults to benjiman.testnet
+//numKeys default is 10, deposit defalt is 10, default drop config 1 use per key and delete on empty, metadata empty, funderBalance false
+async function createFCDrop({funderBalance = false}){
+    //USER'S RESPONSIBILITY TO CHANGE DEFAULT CONSTS IN CONFIGURATIONS.JS
+
+    //init keypom, this takes care of the new NEAR connection
+    console.log("Initiating NEAR connection");
+    initKeypom({network: NETWORK_ID, funder: FUNDER_INFO});
+
+    //create drop, this generates the keys based on the number of keys passed in and uses funder's keypom balance if funderBalance is true (otherwise will sign a txn with an attached deposit)
+    const {keys} = createDrop({
```

Comment:
We'll need to create an NFT series here with metadata etc.. otherwise it will try to mint from a series that doesn't exist

Reply:
used `nft.examples.testnet` instead to avoid needing to create a series. This consequently forces NFT and FC deploy scripts to stricly run on testnet. 

Made it so, if the user wishes to do so, can change the `NFT_CONTRACT_ID` and networks etc. to utilize mainnet if they wish to do so. The output json will reflect that accordingly.